src/search/heuristics/cea_heuristic.cc

```diff
@@ -450,6 +454,12 @@ class ContextEnhancedAdditiveHeuristicFeature
     ContextEnhancedAdditiveHeuristicFeature() : TypedFeature("cea") {
         document_title("Context-enhanced additive heuristic");
 
+        add_option<bool>(
```

Comment:
I saw this same text in multiple places. For other cases like this, we introduced two static methods to add an option and to parse the arguments from an options object. This way, you only have the documentation in one place,

For example for log objects:
```c++
extern void add_log_options_to_feature(plugins::Feature &feature);
extern std::tuple<Verbosity> get_log_arguments_from_options(
    const plugins::Options &opts);
```

Reply:
You might also be able to move the logic of `get_default_value_axioms_task_if_needed` there to avoid more code duplication.

Reply:
I added the two methods to the `default_value_axioms_task.{cc,h}` files and call it on the relevant places. At the same time I changed the option from a bool to an enum after today's discussion with Gabi and Malte.

I don't understand how to move the logic of `get_default_value_axioms_task_if_needed`. I would imagine that this needs to be a separate method since we need to pass its return value to the constructor of `Heuristic`, but maybe I'm overlooking something?

Reply:
We talked about this offline and decided there is nothing to do.